include/UnicycleGenerator.h

```diff
@@ -53,7 +53,7 @@ class UnicycleGenerator {
 
     bool setTerminalHalfSwitchTime(double lastHalfSwitchTime); //if not set, it won't bring the ZMP at the center of the feet at the end
 
-    bool setPauseConditions(double maxStepTime, double nominalStepTime);
+    bool setPauseConditions(double maxStepTime, double nominalStepTime, bool pauseActive = false);
```

Comment:
To be honest I don't like these for two reasons:
- It is ``false`` by default, meaning that it changes the behavior of this piece of code without being visible. What if some other code used this function before? After updating the behavior would be different without any apparent modification.
- It is not very clear when reading the code. Also, why would you set the pause conditions and deactivate them in the same method?

I would suggest adding a method like
```c++
void disablePauseConditions()
```
to disable pause conditions. On the contrary, the method ``setPauseConditions`` should enable them automatically.

Reply:
> I would suggest adding a method like
> void disablePauseConditions()

This modification done in this commit. https://github.com/robotology/unicycle-footstep-planner/pull/34/commits/e5704c2d2c7652b793751e9aa2c4be6f53e5957c

> to disable pause conditions. On the contrary, the method setPauseConditions should enable them automatically.

Has it already been implemented ?

Reply:
> Has it already been implemented ?

I think so: https://github.com/robotology/unicycle-footstep-planner/blob/fd10aae80e244feeba8f8c669181d1cad4e7f9a2/src/UnicycleGenerator.cpp#L590